weblab/conftest.py

```diff
@@ -20,9 +23,30 @@ def add_version(entity, filename='file1.txt', tag_name=None):
 
 @pytest.fixture
 def helpers():
+    """
+    Provide helpers in the form of a fixture
+    """
     return Helpers
 
 
+@pytest.fixture(autouse=True)
+def fake_upload_path(settings, tmpdir):
+    settings.MEDIA_ROOT = str(tmpdir)
+    return settings.MEDIA_ROOT
+
+
+@pytest.fixture(autouse=True)
+def fake_experiment_path(settings, tmpdir):
```

Comment:
Something I just thought of - do these separate uses of `tmpdir` get different temporary folders?

Reply:
If that becomes an issue, we can use the `tempdir` module:

- https://docs.python.org/2/library/tempfile.html
- https://github.com/pints-team/pints/blob/master/test/shared.py#L135-L174

Reply:
https://docs.pytest.org/en/latest/tmpdir.html isn't entirely clear, but I think they're separate!

Reply:
Sorry, I meant the `tempfile` module! The other code I posted is just a ContextManager I wrapped around it